src/api.js

```diff
@@ -0,0 +1,28 @@
+import axios from "axios";
+
+
+const apiClient = axios.create({
+  baseURL: "https://news-lzt1.onrender.com/api",
+  timeout: 10000,
```

Comment:
Should we have a timeout? The api is hosted on a free tier which can take a while to spin up so we should be allowing it to load regardless of how long it may take